homeassistant/components/aseko_pool_live/__init__.py

```diff
@@ -48,3 +52,21 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1:
+        new = {
+            CONF_EMAIL: config_entry.title,
+            CONF_PASSWORD: "",
+        }
+
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(config_entry, data=new)
+
+    _LOGGER.debug("Migration to version %s successful", config_entry.version)
+
+    return True
```

Comment:
We should return `False` in the default case, i.e. if trying to downgrade from a future version.
```suggestion
        _LOGGER.debug("Migration to version %s successful", config_entry.version)
        return True
        
    _LOGGER.error("Attempt to migrate from unknown version %s", config_entry.version)
    return False
```